modules/hcp-bm-firewall-port-svc-reqs.adoc

```diff
@@ -15,6 +15,8 @@ Services run on their default ports. However, if you use the `NodePort` publishi
 
 Use firewall rules, security groups, or other access controls to restrict access to only required sources. Avoid exposing ports publicly unless necessary. For production deployments, use a load balancer to simplify access through a single IP address.
 
+If your hub cluster has a proxy configuration, ensure that it can reach the hosted cluster API endpoint either through the proxy or by updating the `no_proxy` values on the hub cluster. For more information, see "Configuring the cluster-wide proxy".
```

Comment:
update the `no_proxy` value with what?

Reply:
> @CrystalChun, should `no_proxy` be `noProxy`?
Sorry, yes!

As for the value, let's clarify that any and all hosted cluster API endpoint(s) are added to the `noProxy` value

Reply:
@zanetworker @CrystalChun Does this sound correct?
> If your hub cluster has a proxy configuration, ensure that it can reach the hosted cluster API endpoint by adding all hosted cluster API endpoints to the `noProxy` field on the `Proxy` object. For more information, see "Configuring the cluster-wide proxy".

Reply:
Sounds good to me!